app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -93,7 +93,8 @@ import {
 import { CUSTOM_USER_SEARCH_OPTIONS } from "select-kit/components/user-chooser";
 import { downloadCalendar } from "discourse/lib/download-calendar";
 
-// If you add any methods to the API ensure you bump up this number
+// If you add any methods to the API ensure you bump up the version number
+// based on Semantic Versioning 2.0.0.
```

Comment:
I think we should be more explicit about this. I've often seen the patch version being bumped when we're adding new features in a backwards compatible manner.